newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -117,8 +126,12 @@ const ExtensionInstallDialog = ({
             label={
               !isCompatible ? (
                 <Trans>Not compatible</Trans>
-              ) : alreadyInstalled ? (
```

Comment:
Any mention of alreadyInstalled in the dialog should be replaced by a boolean infered from what is returned by the hook I guess.